capstone/src/main/webapp/index.html

```diff
@@ -8,6 +8,9 @@
     <link rel="stylesheet" href="styles/login.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
     <script src="scripts/auth.js"></script>
+    <script
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCofxonKnl8CaGEMd0Ppr_onhfom2HDMm0"
+    ></script>
```

Comment:
This looks like we are committing API keys to github, which we do not want to do for security reasons - keys are handed out on a per person basis, and if someone knows your key then they could make requests on your behalf without you knowing. Additionally, Google's Security team reached out to the intern team to remind everyone to protect the API keys. 
These were their suggestions-- 
Please be sure to keep these credentials private. A few ways to do this include:
- Using a .gitignore file to exclude the file containing the service account credentials.(i.e. including /path/to/api_key.json in the .gitignore file)
- Place the file with the API key outside of the Github project and retrieve it by creating an environment variable (i.e. GCP_API_KEY=/path/to/api_key.json).

My suggestion is making an api_key.json file for all API keys that y'all need, add that file to the .gitignore, and then make sure everyone has that file so they can build/deploy the app with the appropriate API keys. It does make this a little more complicated in that in JS you'll need to create a script tag and then set the src to be the 
`"https://maps.googleapis.com/maps/api/js?key=" + apiKey` where apiKey comes from the .json file